src/themes/teams/components/Icon/svg/ProcessedIcons/icons-accept.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g>
+        <path
+          className={classes.outlinePart}
```

Comment:
In order to support changing visible paths on parent element's hover (#364), I will need to add static class names to both outline and filled paths. I plan to change this to something like:
```jsx
className={iconOutlinePart(classes)}
```
Please let me know if that works for you, I will do that in a separate PR once this one is merged.

Reply:
As long as the filledPart & outlinePart in IconStyles still work, then we're good.